src/Compilers/Core/AnalyzerDriver/AnalyzerManager.cs

```diff
@@ -265,12 +265,6 @@ internal bool IsSupportedDiagnostic(DiagnosticAnalyzer analyzer, Diagnostic diag
 
             foreach (var diag in supportedDiagnostics)
             {
-                if (HasNotConfigurableTag(diag.CustomTags))
```

Comment:
nope, this doesn't make diagnostic configurable. It just means that this diagnostic now contributes in whether or not analyzer is suppressed. eventually any diagnostic reported with ID for non-configurable diagnostic will have default severity.